programmers/짝수_홀수.js

```diff
@@ -0,0 +1,8 @@
+function solution(i) {
+
+    if (i % 2){
+    return i = 'Odd';
+    }
+    else {return i = 'Even'}
```

Comment:
```suggestion
    return 'Odd';
    }
    return 'Even';
```
굳이 변수에 안담아도 되는 경우에는 그냥 return 해주셔도 됩니다.